packages/cozy-sharing/src/SharingProvider.jsx

```diff
@@ -196,6 +197,7 @@ export class SharingProvider extends Component {
     )
 
     this.dispatch(receivePaths([...folderPaths, ...filePaths]))
+    this.setState({ allLoaded: true })
```

Comment:
This commit should be squashed with the latest one. This is "buggy" commit since this commit doesn't do what we want to.